docs/configuration.asciidoc

```diff
@@ -564,6 +564,10 @@ When set to `true` the agent will capture stack traces for spans whose duration
 
 A time value to control whether the agent will report stack traces for measured spans. If a span's duration reaches this threshold, the agent will report stack traces. If <<capture-span-stack-traces, `captureSpanStackTraces`>> is set to `false` the agent will ignore the `spanFramesMinDuration` value.
 
+When set to `0` the agent will not record stack traces for any spans.
+
+When set to a negative value (like `-1`) the agent will always record stack traces for spans.
```

Comment:
```suggestion
When set to a negative value (like `-1ms`) the agent will always record stack traces for spans.
```

Should include units so it isn't ambiguous. (Though, granted that in this case for a negative value compared against duration that the units don't matter.)